server/src/main/java/com/flipkart/varadhi/web/v1/VaradhiEntityHandlers.java

```diff
@@ -0,0 +1,78 @@
+package com.flipkart.varadhi.web.v1;
+
+import com.flipkart.varadhi.auth.PermissionAuthorization;
+import com.flipkart.varadhi.entities.*;
+import com.flipkart.varadhi.services.VaradhiEntityService;
+import com.flipkart.varadhi.services.VaradhiEntityServiceFactory;
+import com.flipkart.varadhi.utils.VaradhiEntityUtils;
+import com.flipkart.varadhi.web.Extensions.RequestBodyExtension;
+import com.flipkart.varadhi.web.Extensions.RoutingContextExtension;
+import com.flipkart.varadhi.web.routes.RouteDefinition;
+import com.flipkart.varadhi.web.routes.RouteProvider;
+import com.flipkart.varadhi.web.routes.SubRoutes;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.ext.web.RoutingContext;
+import lombok.experimental.ExtensionMethod;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import static com.flipkart.varadhi.auth.ResourceAction.*;
+import static com.flipkart.varadhi.web.routes.RouteBehaviour.authenticated;
+import static com.flipkart.varadhi.web.routes.RouteBehaviour.hasBody;
+
+@Slf4j
+@ExtensionMethod({RequestBodyExtension.class, RoutingContextExtension.class})
+public class VaradhiEntityHandlers implements RouteProvider {
+
+    private final VaradhiEntityServiceFactory varadhiEntityServiceFactory;
+
+    public VaradhiEntityHandlers(VaradhiEntityServiceFactory varadhiEntityServiceFactory) {
+
+        this.varadhiEntityServiceFactory = varadhiEntityServiceFactory;
+
+    }
+
+    @Override
+    public List<RouteDefinition> get() {
+        //TODO : implement this for each entity type
+        return new SubRoutes(
+                "/v1/tenants/:tenant/orgs",
```

Comment:
post will be on /v1/orgs with body containing the name of the org and get/put/delete will be on /v1/orgs/:org